### DISCUSSION TITLE
Reading multiple geojson files from a folder

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#337] [Ideas]
Greetings

Is there a way to use this plugin to read and aggregate multiple geojson files in a folder. Right now infinity does url but is there any future plans for reading from disk?

Thanks!

**Comment by yosiasz** [comment]
or reading any file from disk 😉

**Reply by yesoreyeram** [reply]
File reading is intentionally disabled. For obvious security reasons.
